src/vs/base/browser/ui/hover/hover.css

```diff
@@ -131,3 +131,8 @@
 	border-bottom: 1px solid transparent;
 	text-underline-position: under;
 }
+
+.monaco-hover .markdown-hover .hover-contents span {
+	margin-bottom: 4px;
```

Comment:
Can you add a comment about why this is here?

Reply:
Added!

Reply:
Given that almost all of the other sizes are `px`, I think it is safe to keep it `px`.